### DISCUSSION TITLE
Selling Watermeter with installed AI-on-the-edge?

### DISCUSSION THREAD

**Discussion by rzucki71** [jomjol/AI-on-the-edge-device#3386] [General]
I did built a quite comfortable watermeter housing, which I would like to offer **commercially**.
Either for those without 3D-Printer (only housing), or for with missing technical skills (housing including ESP32-CAM, USB-Connector but also WLAN-Antenna and more).

My question is (because I could not find any licensing Information):
Is it **legally** possible to offer this housing as a "complete-Set" (with "AI-on-the-edge" flashed ESP including files on SD-Card)?
This would be **without** any extra profit!

kind regards!

**Comment by rzucki71** [comment]
![IMG_20241116_142405](https://github.com/user-attachments/assets/3a0e1bc4-af32-41c8-bb80-cdd8175a584d)

**Reply by Mithrox** [reply]
If you were to do something like that I would absolutely like to try that, the challenge I guess would be for you to get the exact measurements of the different water gauges there are, like mine from Norway may look different than other countries.

**Comment by jomjol** [comment]
Hi,
there is no license by purpouse. Please send me an email and we can discuss the licensing (mueller.josef(a)gmail.com)
Beste regards,
  jomjol
